driver/test/java/io/crate/client/jdbc/integrationtests/CrateJDBCConnectionTest.java

```diff
@@ -129,8 +129,14 @@ private static void insertIntoTable() throws SQLException {
         preparedStatement.setObject(10, objectField);
         preparedStatement.setString(11, "1970-01-01");
         preparedStatement.setString(12, "127.0.0.1");
-        preparedStatement.setArray(13, connection.createArrayOf("varchar", new String[]{"a", "b", "c", "d"})); // TODO array support
-        preparedStatement.setArray(14, null);
+        preparedStatement.setArray(13, connection.createArrayOf("varchar", new String[]{"a", "b", "c", "d"}));
```

Comment:
the usage must also be documented.
https://github.com/crate/crate-jdbc/blob/master/docs/index.txt#L209-L230